src/neptune/internal/streams/std_stream_capture_logger.py

```diff
@@ -67,7 +70,7 @@ def work(self) -> None:
                 data = self._logger._log_data_queue.get()
                 if data is None:
                     break
-                self._logger._logger.log(data)
+                self._logger.log_data(data)
```

Comment:
what is `self._logger` pointing to? 
You removed line `self._logger = NeptuneLogger(container, attribute_name)` so I seems like a None

Reply:
`self._logger` still points to the same thing. What I removed is `self._logger._logger` ;)

Reply:
yep ur right